_sass/_layout.scss

```diff
@@ -57,12 +57,19 @@
 
         // Gaps between nav items, but not on the first one
         &:not(:first-child) {
-            margin-left: 20px;
+            margin-left: 12px;
         }
 
         &.home-link {
```

Comment:
This should get removed. We're not using it anymore.

Reply:
done.

Reply:
actually, it turns out we are. I've reinstated the style. we should discuss how to best approach cleaning up CSS / SCSS here.

Reply:
I just added `&.home-link` yesterday and it is only used in one place and was replaced with `space-link` in this change.  So it shouldn't be used anywhere else.

Reply:
space-link does not replace home-link; it provides a different case state for use with page-link. they are both still in use separately in the nav.